src/utils/use-hover.ts

```diff
@@ -3,10 +3,13 @@ import { useState } from 'react'
 /**
  * A simple React Hook that responds to `hover` events.
  */
-const useHover = () => {
-  const [hovered, setHovered] = useState<boolean>(false)
+const useHover = (): [
+  boolean,
+  { onMouseEnter: () => any; onMouseLeave: () => any },
+] => {
+  const [isActive, setHovered] = useState<boolean>(false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)